tests/phpunit/tests/admin/wpSiteHealth.php

```diff
@@ -499,4 +499,80 @@ public function data_object_cache_thresholds() {
 			array( 'alloptions_bytes', 1000 ),
 		);
 	}
+
+	/**
+	 * Tests get_test_autoloaded_options() when autoloaded options less than warning size.
+	 *
+	 * @ticket 61276
+	 *
+	 * @covers ::get_test_autoloaded_options()
+	 */
+	public function test_wp_autoloaded_options_test_no_warning(): void {
+		$expected_label  = esc_html__( 'Autoloaded options are acceptable' );
+		$expected_status = 'good';
+
+		$result = $this->instance->get_test_autoloaded_options();
+		$this->assertSame( $expected_label, $result['label'] );
+		$this->assertSame( $expected_status, $result['status'] );
```

Comment:
Test methods with multiple assertions should have a final `$message` argument passed to each assertion to provide context to each failure. [Ref](https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions).

:1234: Also applies elsewhere in the PR

Reply:
Updated. Thanks!